### PR TITLE
Add disable badge counter

### PR DESCRIPTION
**Pull request by vrnvorona**
https://community.bitwarden.com/t/disable-badge-counter/18377

### CODE REVIEW

**Review by cscharf — CHANGES_REQUESTED** [review]
Thanks for the submission!

Outside of the `constants.service` for the constants key, I would be inclined to say this is specific to the `browser` project only, and all implementation of loading/saving this setting should be encapsulated in there; favicons are relevant to all clients who load these icons for vault entries whereas the badge counter is only used in the browser extension.

These are located in browser/src/popup/accounts/*.component.ts

File: src/angular/components/send/add-edit.component.ts [hunk 1]
```diff
@@ -51,7 +51,6 @@ export class AddEditComponent implements OnInit {
     expirationDateSelect: number = null;
```
Comment (eliykat) [inline src/angular/components/send/add-edit.component.ts:51]: Can you remove these unrelated changes to this file?

Comment (vrnvorona) [reply src/angular/components/send/add-edit.component.ts]: Done, accidentally reverted it earlier

**Review by cscharf — APPROVED** [review]
